native/src/components/PoiDetails.tsx

```diff
@@ -99,9 +99,7 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
         )}
         {distance && (
           <PoiDetailItem icon='place' language={language} onPress={detailPage ? navigateToPois : undefined}>
-            <Text>
-              {distance} {t('unit')} {t('distanceText')}
-            </Text>
+            <Text>{t('distanceKilometer', { distance })}</Text>
```

Comment:
```suggestion
            <Text>{t('distanceKilometre', { distance })}</Text>
```
would like to keep stuff en. but not important. if then pls also adjust translation key